system-int-test/test/cmr/system_int_test/ingest/subscription_processing_test.clj

```diff
@@ -115,9 +108,9 @@
      (testing "Deleting the subscription purges the suscription notification entry"
        ;; Delete and reingest the subscription. If the sub-notification was purged, then it
        ;; should look back 24 hours, as if the subscription were new.
-       (ingest/delete-concept subscription-concept)
-       (subscription-util/ingest-subscription subscription-concept
-                                              {:token "mock-echo-system-token"})
+       (let [concept {:provider-id "PROV1" :concept-type :subscription :native-id "test_sub_prov1"}]
+         (ingest/delete-concept concept))
+       (subscription-util/create-subscription-and-index coll1, "test_sub_prov1" "user2" "provider=PROV1")
```

Comment:
unnecesary comma

Reply:
Removed here, and a few other places where `create-subscription-and-index` was called